### PR TITLE
Fix/add one line quotation display

### PR DESCRIPTION
**Pull request by zdzislawbela**

### CODE REVIEW

File: events/handlePressedButton.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    
+    if (modifier === '+/-') {
+        if (currentValue.slice(0,1) === '-') {
+            currentValue = currentValue.slice(2,currentValue.length-1);
+        } else {
+            currentValue = `-(${currentValue})`;
+        }  
+    }
+    if (modifier == '.') {
+        if (currentValue.slice(currentValue.length-1,currentValue.length) === ")") {
```
Comment (raqso) [inline events/handlePressedButton.js:47]: extarct to const

File: events/handlePressedButton.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+            currentValue = `${currentValue}.`;
+        } 
+    }
+    updateDisplay(savedValues, currentValue);
+}
+
+const userPressedOperator = (operator) => {
+
+    if(
+        parseInt(currentValue.slice(currentValue.indexOf(".")+1, currentValue.length)) === 0
```
Comment (raqso) [inline events/handlePressedButton.js:62]: :hankey:

File: events/handlePressedButton.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+        currentValue = "0";
+    }
+    updateDisplay(savedValues, currentValue);
+}
+
+const userPressedEqualSign = () => {
+    let sum = "";
+    const savedCleanedValues = [];
+
+    savedValues.forEach((savedValue) => {
```
Comment (raqso) [inline events/handlePressedButton.js:94]: Put into map

File: events/handlePressedButton.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+        operatorIndex1 === "+" ? sum = numberIndex0 + numberIndex2: "";
+        operatorIndex1 === "-" ? sum = numberIndex0 - numberIndex2: "";
+    }
+
+    console.log(`calculatedValues: ${calculatedValues}`)
+    console.log(`sum: ${sum}`)
+    const newDisplayRow = document.createElement('div');
+    newDisplayRow.setAttribute('class', 'displayRow');
+    newDisplayRow.innerHTML = sum;
+    document.querySelector('.display').appendChild(newDisplayRow);
```
Comment (raqso) [inline events/handlePressedButton.js:126]: Extract to function

File: src/calculator/keyboard/keyboard.css [hunk 5]
```diff
@@ -20,6 +21,20 @@
     border-style: solid;
     border-color: black;
 }
+/* .operator
```
Comment (raqso) [inline src/calculator/keyboard/keyboard.css:24]: 🗑️

File: src/calculator/keyboard/keyboard.js [hunk 6]
```diff
@@ -1,107 +1,27 @@
 import classes from './keyboard.css';
 
 const keyboard = () => {
-
     const keyboard = document.createElement('div');
     keyboard.setAttribute('class', 'keyboard');
     document.querySelector('.calculator').appendChild(keyboard);
-
+    const ButtonClass = "button";
```
Comment (raqso) [inline src/calculator/keyboard/keyboard.js:7]:
```suggestion
    const buttonClass = "button";
```

File: src/calculator/keyboard/keyboard.js [hunk 7]
```diff
@@ -1,107 +1,27 @@
...
+    
     const buttons = [
```
Comment (raqso) [inline src/calculator/keyboard/keyboard.js:9]: Extract to a enum

File: src/main.js [hunk 8]
```diff
@@ -1,10 +1,9 @@
 import calculator from './calculator/calculator.js';
-import {setKeyClassInterval} from './handleEvents.js';
-import {addKeyListeners, addMathKeyListeners} from './keyListeners.js';
+//import {setKeyClassInterval} from '../events/displayEvents.js';
```
Comment (raqso) [inline src/main.js:2]: 🗑️

File: src/main.js [hunk 9]
```diff
@@ -1,10 +1,9 @@
...
+import {buttonsListener} from '../events/keyListeners.js';
 
 export default () => {
     calculator();
-    addKeyListeners();
-    addMathKeyListeners();
-    setInterval(setKeyClassInterval, 3000);
+    buttonsListener();
+    //setInterval(setKeyClassInterval, 3000);
```
Comment (raqso) [inline src/main.js:8]: 🗑️

File: src/main.js [hunk 10]
```diff
@@ -1,10 +1,9 @@
 import calculator from './calculator/calculator.js';
-import {setKeyClassInterval} from './handleEvents.js';
-import {addKeyListeners, addMathKeyListeners} from './keyListeners.js';
+//import {setKeyClassInterval} from '../events/displayEvents.js';
+import {buttonsListener} from '../events/keyListeners.js';
 
 export default () => {
```
Comment (raqso) [inline src/main.js:5]: Name the function

**Review by raqso — CHANGES_REQUESTED** [review]

File: events/eventHandlers_old.js [hunk 11]
```diff
@@ -1,7 +1,38 @@
...
+const numbers = [];
+const operators = [];
+
+const storeNumber = () => {
+    console.log("Numbers: " + numbers);
+    const lastRow = document.querySelectorAll('.displayRow');
+    const lastRowValue = lastRow[lastRow.length - 1].innerHTML;
+    console.log("lastRowValue:" + lastRowValue);
+
+    //const lastRowVaule = "3.33+1.223";
```
Comment (zdzislawbela) [inline events/eventHandlers_old.js:12]:
```suggestion
    //const lastRowVaule = "3.33+1.223";
    
```

File: events/displayEvents.js [hunk 12]
```diff
@@ -0,0 +1,30 @@
+
+const updateDisplay = (savedDisplayValues, currentDisplayValue) => {
+
+    const displayRow = document.querySelectorAll('.displayRow');
+    const lastDisplayRow = displayRow[displayRow.length - 1];
+    let displayValues = ""
+
+    if (savedDisplayValues) {
+        savedDisplayValues.forEach((savedValue) => {
```
[the remark this answers is not in the document]
Comment (zdzislawbela) [reply events/displayEvents.js:9]: but I need a string not an array

**Pull request merged**